src/components/MenuItem.tsx

```diff
@@ -568,6 +568,15 @@ function MenuItem(
         }
     };
 
+    const getSubscriptpAvatarBackgroundColor = (isHovered: boolean, pressed: boolean, hoveredBackgroundColor: string, pressedBackgroundColor: string) => {
+        if (pressed) {
+            return pressedBackgroundColor;
+        }
+        if (isHovered) {
+            return hoveredBackgroundColor;
+        }
```

Comment:
NAB. Since we are passing the colors as parameters we should move this function outside the component